actions/retest/main.go

```diff
@@ -176,6 +176,20 @@ func main() {
 							log.Printf("failed to create comment %v\n", err)
 							continue
 						}
+						found = true
+					}
+				}
+
+				if found {
+					// comment `@Mergifyio refresh` so mergifyio adds the pr back into the queue.
+					msg := "@Mergifyio refresh"
```

Comment:
add this comment only if all checks are passed on the PR.

Reply:
@Madhu-1  This will be added after all the failed tests have been restarted ( there should atleast be a single failed test for this scenario to occur).

Reply:
but what is the use if the test is not successful, this will block the queue as it added but failed?